bundles.ini

```diff
@@ -123,6 +136,20 @@ packages    = osg-wn-client osg-update-data osg-ca-scripts
 repofile    = repos/osg-3.5-%(dver)s.repo.in
 stage1file  = osg-stage1-%(dver)s.lst
 
+[osg-afs-client-3.6]
+paramsets   = el7,x86_64
+patchdirs   = patches/wn-client/common
+              patches/wn-client/common/%(dver)s
+              patches/wn-client/3.6/%(dver)s
+              patches/afs-client/3.6/common
+              patches/afs-client/3.6/%(dver)s
+dirname     = osg-afs-client
+tarballname = osg-afs-client-%(version)s-%(relnum)s.%(dver)s.%(basearch)s.tar.gz
+packages    = osg-wn-client osg-update-data osg-ca-scripts
```

Comment:
I think we don't have osg-update-data.

Reply:
Never mind, it's part of osg-update-vos.

Reply:
We should, does it have to be in release or something? https://koji.chtc.wisc.edu/koji/buildinfo?buildID=14298